opal/mca/btl/ofi/btl_ofi_component.c

```diff
@@ -213,6 +226,20 @@ void mca_btl_ofi_exit(void)
     exit(1);
 }
 
+static int is_in_list(char * const *list, const char *item)
+{
+    int i;
+
+    if (!list)
```

Comment:
Please use Open MPI style of putting `{}` around *all* blocks, even 1-line blocks (we have very few style restrictions, but this is one of them; see https://github.com/open-mpi/ompi/wiki/CodingStyle).